cohortextractor/dsl.py

```diff
@@ -0,0 +1,185 @@
+"""This module provides classes for building a cohort using the DSL.
+
+Data lives in database tables, which are represented by PatientTable (a table with one
+row per patient) and EventTable (a table with multiple rows per patient).
+
+Through filtering, sorting, aggregating, and selecting columns, we transform instances
+of PatientTable/EventTable into instances of PatientSeries.
```

Comment:
This rewrite of the comment and diagram has lost the explicit statement that a `*Table` _is a_ `*Series`.

Reply:
I think you mean `*Frame` -- but yes, will address.

Reply:
Addressed in ffee5b6ce57594ee7ea10f7c2c2a090c4499dfd6.